test/suite/lib.rs

```diff
@@ -369,6 +369,62 @@ pub async fn test_tx_deduplication(ledger_env: impl LedgerEnv) -> anyhow::Result
     Ok(Outcome::Passed)
 }
 
+// Checks whether the errors returned by the icrc1_transfer endpoint are used correctly
+pub async fn test_transfer_errors(ledger_env: impl LedgerEnv) -> anyhow::Result<Outcome> {
+    // Create two test accounts and transfer some tokens to the first account
+    let p1_env = setup_test_account(&ledger_env, 200_000.into()).await?;
+    let p2_env = p1_env.fork();
+
+    let mut transfer_args = Transfer::amount_to(10_000, p2_env.principal());
+    let ledger_fee = transfer_fee(&ledger_env).await.unwrap();
+    // Set incorrect fee
```

Comment:
Let's not unwrap any errors that can realistically happen.
As I mentioned before, any panic will kill the entire test suite, not just a single test.